Test deployment

WNjihia opened the pull request:
Closes #3 
After testing that the deployment works, this branch updates the deployment instructions on the README

dmkitui commented on README.md:
````diff
@@ -74,38 +74,86 @@ docker push codeforafrica/ckan:latest
+cd dokku
+```
+
+Assign IP address 10.0.0.2 to dokku.me
````
I feel it will be much clearer to someone if this step was more explicitly outlined, together with maybe a screenshot. 
After running `sudo nano /private/etc/hosts`, the user should add a line to the end of the file with the IP address.
`10.0.0.2        dokku.me`

![Uploading Screen Shot 2017-09-19 at 12.09.13.png…]()

WNjihia replied:
Great suggestion. Thanks, I will add this

dmkitui replied:
Great.

DavidLemayian commented:
Thanks for this @WNjihia. Deployment would assume having a linux server to achieve this so setting up on Mac would be a development process instead of deployment.

For local development, one can create a VM as they best see fit. We can flag this in the documentation but not be prescriptive about it.

Let's therefore focus on deployment docs after one has dokku set up. And also not be prescriptive about letsencrypt or other items but instead reference docs to the plugins etc. and be minimal about the deployment here.

WNjihia commented:
@DavidLemayian I have edited the README.md and removed the instructions on setting up on a Mac. I have also added reference docs to the plugins and omitted the prescriptive details about the plugins

DavidLemayian commented on README.md:
````diff
@@ -74,38 +74,55 @@ docker push codeforafrica/ckan:latest
 git push dokku
 ```
 
+***NOTE:** Make sure to have the [appropraite permissions to push to dokku](?).*
````
Please add link for dokku permission where "?" is.

WNjihia replied:
Done

DavidLemayian commented on README.md:
```diff
@@ -74,38 +74,55 @@ docker push codeforafrica/ckan:latest
 ## License
 
-MIT 
+MIT <!-- Should be more... -->
```
Let's also do this while we're here.

WNjihia replied:
Done
